src/NHibernate/IStatelessSession.cs

```diff
@@ -31,6 +31,23 @@ public static IQueryBatch CreateQueryBatch(this IStatelessSession session)
 		/// <returns>The current transaction or <see langword="null" />..</returns>
 		public static ITransaction GetCurrentTransaction(this IStatelessSession session)
 			=> session.GetSessionImplementation().ConnectionManager.CurrentTransaction;
+
+		//NOTE: Keep it as extension
+		/// <summary>
+		/// Return the persistent instance of the given entity name with the given identifier, or null
+		/// if there is no such persistent instance. (If the instance, or a proxy for the instance, is
+		/// already associated with the session, return that instance or proxy.)
+		/// </summary>
+		/// <typeparam name="T">The entity class.</typeparam>
+		/// <param name="session">The session.</param>
+		/// <param name="entityName">The entity name.</param>
+		/// <param name="id">The entity identifier.</param>
+		/// <param name="lockMode">The lock mode to use for getting the entity.</param>
+		/// <returns>A persistent instance, or <see langword="null" />.</returns>
+		public static T Get<T>(this IStatelessSession session, string entityName, object id, LockMode lockMode = null)
```

Comment:
Probably split this into 2 methods to be consistent with ISession.

Reply:
Done